test/cmd/dnshammer.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"sort"
+	"time"
+
+	"github.com/rancher-sandbox/rancher-desktop-host-resolver/test/testdns"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const lookupTimeoutSec = 10 // 10s timeout should be adequate
```

Comment:
I think this would be clearer expressed as:
```suggestion
const lookupTimeout = 10 * time.Second
```
(Also, the comment doesn't really say anything useful.)